src/util.js

```diff
@@ -4,7 +4,7 @@ const CID = require('cids')
 const protons = require('protons')
 const proto = protons(require('./dag.proto.js'))
 const DAGLink = require('./dag-link')
-const DAGNode = require('./dag-node')
+const DAGNode = require('./dag-node/dagNode.js')
```

Comment:
Same as above, please remove the `.js`.

In regards to the line above, would it make sense to change it to:

```JS
   const DAGLink = require('./dag-link/dagLink')
```

for more consistency?